README

```diff
@@ -888,9 +888,9 @@ IMGMOUNT
         none: DOSBox will make no attempt to read the file system on the disk.
               This is useful if you need to format it or if you want to boot
               the disk using the BOOT command. When using the "none"
-              filesystem, you must specify the drive number (2 or 3,
-              where 2 = master, 3 = slave) rather than a drive letter.
-              For example, to mount a 70MB image as the slave drive device,
+              filesystem, you must specify the drive number: 2 or 3, where
+              two is primary and three is secondary, rather than a drive letter.
+              For example, to mount a 70MB image as the secondary drive device,
```

Comment:
>     In parallel ATA hard drive arrangements, the terms master and slave are used to describe drives on the same cable, but neither drive has control or priority over the other.

Good place to remove misleading master/slave terminology. However, in this case it referred to old designation of master/slave as seen in BIOS - so at least people familiar with old PCs understood it (and probably noone else). Primary/secondary works better, but I think we need to rewrite the sentence to make it less "awkward" now.

For non-technical person it might be weird that drive number 2 is "primary" and drive number 3 is "secondary". Maybe we can put a short list instead? Something like:

```
Drive numbers:
0 - primary floppy drive
1 - secondary floppy drive
2 - primary hard drive
3 - secondary hard drive
```

Reply:
Very nice @dreamer ; it reads very natural now. fixup! inbound.